src/main/java/frc/robot/utility/math/DoubleInterpolation.java

```diff
@@ -0,0 +1,47 @@
+package com.team254.lib.util;
```

Comment:
Wrong package

Reply:
I don't need the DoubleInterpolation for anything, it just wanted to be part of the commit for some reason. The class was filled with errors (more than just the package), so I deleted it.